### PR TITLE
refactor(mc-script): compile it through babel, to support node 10

### PR DESCRIPTION
**Pull request by emmenko**
This PR addresses the issue described in #1466 for apps using an old node version.

TL;DR: compile the code with Babel.

Other PRs will follow to migrate the other "nodejs" packages.

### CODE REVIEW

File: packages/mc-scripts/README.md [hunk 1]
```diff
@@ -4,10 +4,22 @@
...
+## Usage
+
+The package exports some functions to configure Webpack
+
+```js
+const {
+  createWebpackConfigForDevelopment,
+  createWebpackConfigForProduction,
+  vendorsToCompile,
+} = require('@commercetools-frontend/mc-scripts');
```
Comment (emmenko) [inline packages/mc-scripts/README.md:24]: This is new, but there is proxy `config` folder for backwards compatibility.

File: packages/mc-scripts/package.json [hunk 2]
```diff
@@ -79,8 +93,26 @@
...
+        {
+          "targets": {
+            "node": "10"
+          },
+          "modules": "commonjs",
+          "useBuiltIns": "usage",
+          "corejs": 3
+        }
+      ]
+    ]
```
Comment (emmenko) [inline packages/mc-scripts/package.json:116]: I decided to add the babel config here, so it's co-located with the package.

File: application-templates/starter/webpack.config.prod.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
 const path = require('path');
-const createWebpackConfigForProduction = require('@commercetools-frontend/mc-scripts/config/create-webpack-config-for-production');
+const {
+  createWebpackConfigForProduction,
+} = require('@commercetools-frontend/mc-scripts');
```
Comment (tdeekens) [inline application-templates/starter/webpack.config.prod.js:4]: Is the change in export here a breaking change or do we keep the default export for now?

File: packages/mc-scripts/package.json [hunk 4]
```diff
@@ -79,8 +93,26 @@
     "webpack-dev-server": "3.10.3",
     "webpackbar": "4.0.0"
   },
+  "devDependencies": {
+    "@babel/preset-env": "7.9.5"
+  },
   "engines": {
-    "node": ">=8",
+    "node": ">=10",
```
Comment (emmenko) [inline packages/mc-scripts/package.json:100]: Not 100% sure about this, but we should determine a base line most likely.

File: packages/mc-scripts/README.md (hunk 1 above)
Comment (tdeekens) [reply packages/mc-scripts/README.md:24]: Ah, thanks. Disregard my comment regarding that in the review then.

**Pull request merged**